yt/visualization/color_maps.py

```diff
@@ -186,8 +191,16 @@ def add_colormap(name, cdict):
         add_colormap(k, cdict)
 
 
-def _extract_lookup_table(cmap_name):
-    cmap = mcm.get_cmap(cmap_name)
+def get_colormap_lut(cmap_id: Union[Tuple[str, str], str]):
```

Comment:
I'd also suggest adding a docstring to this function, esp. because `lut` is a bit cryptic.

```suggestion
def get_colormap_lut(cmap_id: Union[Tuple[str, str], str]) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
```

Reply:
I'm not 100% sure adding such type hints really helps atm, in particular because there is no way to express what shapes to expect, or the relationships between these arrays' respective shapes (but I expect it will be supported by numpy in the future).

Reply:
as for the docstring, I need to admit I don't know what to write.
"lut" in the context means "lookup table", but the only reason this function exists is historical (I think). Do you think adding a similar comment would be valuable ?

Reply:
At least so that one knows what `lut` stands for. At the moment, it is still unclear to me what the function actually achieves.

Reply:
added a comment